src/webhook/webhook.service.ts

```diff
@@ -13,16 +13,25 @@ export class WebhookService {
     private readonly prismaService: PrismaService,
   ) {}
 
-  async sendWebhookWithRecipe(recipe: RecipeResponse): Promise<void> {
-    const webhookURL = process.env.WEBHOOK_URL;
+  async createRecipeWebhook(recipe: RecipeResponse): Promise<void> {
```

Comment:
Why `createRecipeWebhook`? This is very confusing.
In my understanding, you are not creating anything. You are going to send a webhook which was clearly understandable with the previous method name

If you are going to create one method per webhook type, more meaningful name (IMHO) would be sendRecipeCreatedWebhook